src/foam/lib/json/MapParser.java

```diff
@@ -11,27 +11,36 @@
 public class MapParser extends ProxyParser {
   public MapParser() {
     super(
-          new Seq1(2,
+          new Seq1(1,
                    new Whitespace(),
-                   new Literal("{"),
-                   new Repeat(
-                              new Seq2(1, 5,
-                                       new Whitespace(),
-                                       new AnyKeyParser(),
-                                       new Whitespace(),
-                                       new Literal(":"),
-                                       new Whitespace(),
-                                       new AnyParser()),
-                              new Seq0(new Whitespace(), new Literal(","))),
-                   new Whitespace(),
-                   new Literal("}"))
+                   new Alt(
+                      new Seq1(1, new Literal("{"), new Whitespace(), new Literal("}")),
```

Comment:
Can't simply do `Alt` with `Whitespace` and `Repeat` as `Whitespace` matches all chars (even empty string) so a map key/value would never be picked up (and it would be ambiguous with `Repeat`'s first sequence which begins with a `Whitespace`.